playroom.config.js

```diff
@@ -107,7 +107,7 @@ const config = {
                     type: 'asset/resource',
                 },
                 {
-                    test: /\.vanilla\.css$/i,
+                    test: /\.vanilla\.css/i,
```

Comment:
After the upgrade, I was getting a lot of errors when trying to start playroom. It looks like there are some characters being added to the end of the filenames for vanilla css files. 

![MicrosoftTeams-image (1)](https://github.com/Telefonica/mistica-web/assets/25785151/9f735931-82cb-44bc-89b5-a521bc14c396)